docs/yaml/functions/_build_target_base.yaml

```diff
@@ -161,10 +161,10 @@ kwargs:
   install_dir:
     type: str
     description: |
-      override install directory for this file. The value is
-      relative to the `prefix` specified. F.ex, if you want to install
-      plugins into a subdir, you'd use something like this: `install_dir :
-      get_option('libdir') / 'projectname-1.0'`.
+      override install directory for this file. If the value is a relative path,
+      it will be joined to the value of the `prefix` option.
+      F.ex, if you want to install plugins into a subdir, you'd use something
+      like this: `install_dir : get_option('libdir') / 'projectname-1.0'`.
```

Comment:
This change isn't strictly necessary in the context of this PR, although an argument could be made that the rewording is useful regardless? idk

Point being, `install_dir` currently takes arbitrary locations, including the return value of `get_option('sysconfdir')` (typically the absolute value `/etc`) and also random hardcoded strings (why one would write a meson.build like that I do not know...)

So the existing wording "the value is relative to the prefix" applies to `/etc` being relative to prefix, but since it is an absolute value it cannot actually be made relative... and now more paths are *like* sysconfdir.

Eh, I guess the wording change cannot hurt even just to clarify the /etc case.

Maybe instead:
> If the value is a relative path, it is relative to the `prefix` specified.

Reply:
Yeah, I felt that “the value is relative” was wrong and now it was wrong in even more cases so I decided to improve the wording but I am not completely happy with the new wording.

I do not like “is relative” in your suggestion for the same reason. Maybe something like, “it will be considered relative”?